### PR TITLE
Migrate the RHF inputs from gridstudy-app to commons-ui

### PR DESCRIPTION
**Pull request by flomillot**

### CODE REVIEW

File: demo/src/InputsTab.js [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+const formSchema = yup.object().shape({
+    [AUTOCOMPLETE_INPUT]: yup.string(),
+    [TEXT_INPUT]: yup.string(),
+    [SLIDER_INPUT]: yup.number(),
+    [SELECT_INPUT]: yup.string(),
+    [RADIO_INPUT]: yup.string(),
+    [INTEGER_INPUT]: yup.number(),
+    [FLOAT_INPUT]: yup.number(),
+    [CHECKBOX_INPUT]: yup.boolean(),
+    [SWITCH_INPUT]: yup.boolean(),
```
Comment (archendel) [inline demo/src/InputsTab.js:49]:
```suggestion
    [AUTOCOMPLETE_INPUT]: yup.string().nullable(),
    [TEXT_INPUT]: yup.string(),
    [SLIDER_INPUT]: yup.number().nullable(),
    [SELECT_INPUT]: yup.string().nullable(),
    [RADIO_INPUT]: yup.string().nullable(),
    [INTEGER_INPUT]: yup.number().nullable(),
    [FLOAT_INPUT]: yup.number().nullable(),
    [CHECKBOX_INPUT]: yup.boolean(),
    [SWITCH_INPUT]: yup.boolean(),
```

File: demo/src/InputsTab.js [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+const emptyFormData = {
+    [AUTOCOMPLETE_INPUT]: '',
+    [TEXT_INPUT]: '',
+    [SLIDER_INPUT]: null,
+    [SELECT_INPUT]: '',
+    [RADIO_INPUT]: '',
+    [INTEGER_INPUT]: null,
+    [FLOAT_INPUT]: null,
+    [CHECKBOX_INPUT]: null,
+    [SWITCH_INPUT]: null,
```
Comment (archendel) [inline demo/src/InputsTab.js:37]:
```suggestion
    [AUTOCOMPLETE_INPUT]: null,
    [TEXT_INPUT]: '',
    [SLIDER_INPUT]: null,
    [SELECT_INPUT]: null,
    [RADIO_INPUT]: null,
    [INTEGER_INPUT]: null,
    [FLOAT_INPUT]: null,
    [CHECKBOX_INPUT]: false, // or null, but should then be nullable in schema
    [SWITCH_INPUT]: false, // or null, but should then be nullable in schema
```

File: src/components/react-hook-form/utils/constants.js
Comment (archendel) [inline src/components/react-hook-form/utils/constants.js:1]: does not seem to be used anywhere

**Review by archendel — CHANGES_REQUESTED** [review]

File: src/components/react-hook-form/utils/constants.js
Comment (flomillot) [reply src/components/react-hook-form/utils/constants.js:1]: You're right, thanks

File: README.md
[the remark this answers is not in the document]
Comment (flomillot) [reply README.md:1]: It's on purpose, I just fixed the quotes

**Review by archendel — APPROVED** [review]

File: demo/src/InputsTab.js [hunk 3]
```diff
@@ -0,0 +1,177 @@
+import React from 'react';
```
Comment (thangqp) [inline demo/src/InputsTab.js:1]: Missing Copy right text

**Review by thangqp — APPROVED** [review]
OK

**Pull request merged**